stdlib/csv/csv_test.flux

```diff
@@ -0,0 +1,54 @@
+package csv_test
+
+
+import "array"
+import "testing"
+import "csv"
+
+testcase from_raw{
```

Comment:
I'm breaking a very hard rule here about never commenting on whitespace in reviews, but only as an observation because we maintain the formatter. I think there should be a space here. Did this go through a formatter, and it formatted this way, or are we not checking format on flux in CI? I don't think we should change this, but rather file an issue to get lack of flux formatting caught at the CI level. This can be the indicator that it's done.

Reply:
This did go through the formatter (using the LSP). I'll update to make sure I have the newest version. I agree this needs a space. I'll submit a PR to the formatter if it does not add a space.

Reply:
Updated my LSP and it now adds a space so all good.